src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -1,87 +1,24 @@
-import React, { useState, useMemo, useRef, useCallback } from "react";
-import { useHistory } from "react-router-dom";
-import { OverlayTrigger, Popover } from "react-bootstrap";
+import React, { ReactNode } from "react";
 
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faTicketAlt } from "@fortawesome/free-solid-svg-icons";
+import { currentVenueSelectorData, parentVenueSelector } from "utils/selectors";
 
-import firebase from "firebase/app";
-
-import { DEFAULT_PROFILE_IMAGE, PLAYA_VENUE_ID } from "settings";
-import { IS_BURN } from "secrets";
-
-import { UpcomingEvent } from "types/UpcomingEvent";
-
-import {
-  currentVenueSelectorData,
-  parentVenueSelector,
-  radioStationsSelector,
-} from "utils/selectors";
-import { hasElements } from "utils/types";
-import { venueInsideUrl } from "utils/url";
-
-import { useRadio } from "hooks/useRadio";
 import { useSelector } from "hooks/useSelector";
-import { useUser } from "hooks/useUser";
-import { useVenueId } from "hooks/useVenueId";
 import { useFirestoreConnect } from "hooks/useFirestoreConnect";
 
-import { GiftTicketModal } from "components/organisms/GiftTicketModal/GiftTicketModal";
-import { ProfilePopoverContent } from "components/organisms/ProfileModal";
-import { RadioModal } from "components/organisms/RadioModal/RadioModal";
-import { SchedulePageModal } from "components/organisms/SchedulePageModal/SchedulePageModal";
-
-import NavSearchBar from "components/molecules/NavSearchBar";
-import UpcomingTickets from "components/molecules/UpcomingTickets";
-import { VenuePartygoers } from "components/molecules/VenuePartygoers";
-
-import { NavBarLogin } from "./NavBarLogin";
-
 import "./NavBar.scss";
-import * as S from "./Navbar.styles";
 import "./playa.scss";
 
-const TicketsPopover: React.FC<{ futureUpcoming: UpcomingEvent[] }> = (
-  props: unknown,
-  { futureUpcoming }
-) => (
-  <Popover id="popover-basic" {...props}>
-    <Popover.Content>
-      <UpcomingTickets events={futureUpcoming} />
-    </Popover.Content>
-  </Popover>
-);
-
-const ProfilePopover = (
-  <Popover id="profile-popover">
-    <Popover.Content>
-      <ProfilePopoverContent />
-    </Popover.Content>
-  </Popover>
-);
-
-const GiftPopover = (
-  <Popover id="gift-popover">
-    <Popover.Content>
-      <GiftTicketModal />
-    </Popover.Content>
-  </Popover>
-);
```

Comment:
[notes] Moved to `src/components/organisms/WithNavigationBar/WithNavigationBar.tsx`